static/src/js/routes/Search/Search.js

```diff
@@ -48,6 +46,10 @@ import actions from '../../actions'
 import advancedSearchFields from '../../data/advancedSearchFields'
 import Button from '../../components/Button/Button'
 
+const CollectionDetailsHighlightsContainer = lazy(() => import('../../containers/CollectionDetailsHighlightsContainer/CollectionDetailsHighlightsContainer'))
```

Comment:
Do these fit our line length guidelines?

Reply:
nitpick is good. Eslint should throw warnings when line length is exceeded which aren't here